libraries/Microsoft.Bot.StreamingExtensions/OperationException.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    internal class OperationException : Exception
+    {
```

Comment:
<Editorial from John deleted> You probably don't need this.

Reply:
Well, AutoRest throws Http specific exceptions, and so we need an equivalent exception for streaming operation failures.